spring-xd-module/src/main/java/org/springframework/xd/module/support/ModuleUtils.java

```diff
@@ -37,7 +37,11 @@
  */
 public class ModuleUtils {
 
-	private static final AsciiBytes LIB = new AsciiBytes("lib/");
+	private static final String LIB = "lib/";
+
+	public static final String DOT_JAR = ".jar";
+
+	public static final String DOT_ZIP = ".zip";
```

Comment:
`.zip` is a legitimate extension for a CP element. See http://docs.oracle.com/javase/7/docs/technotes/tools/windows/classpath.html. I don't see what we could possibly gain by not accepting them. 

@ericbottard can you clarify the second part of the comment: org.springframework.xd.module.support.ModuleUtils#createModuleClassLoader(org.springframework.core.io.Resource, java.lang.ClassLoader, boolean) doesn't seem to do any filtering of archives of its own.